crates/ledger-types/src/state_accessor.rs

```diff
@@ -19,7 +19,7 @@ pub trait StateAccessor {
     type L1ViewState: IL1ViewState;
 
     /// Type representing a ledger account's state.
-    type AccountState: IAccountState;
+    type AccountState: IAccountState + Clone;
```

Comment:
Should add this `Clone` constraint to the `IAccountState` trait, otherwise we'd have to include `+ Clone` everywhere it's needed.